rs/tests/node/BUILD.bazel

```diff
@@ -70,3 +70,38 @@ system_test(
         "@crate_index//:slog",
     ],
 )
+
```

Comment:
The convention is to place dynamic testnets under [`rs/tests/testnets`](https://github.com/dfinity/ic/tree/master/rs/tests/testnets) so that others can find them easily. But I can also understand you want to have something more explicitly belonging to the node team so perfectly fine to have it here.